licenses/apache-2.0 - apache-2.0.html

```diff
@@ -0,0 +1,441 @@
+<!DOCTYPE html>
```

Comment:
Need to update license-merge so this apache license is treated the same as the other one

Reply:
I've removed the dependencies that caused this :)